docs/guides/development.md

```diff
@@ -85,64 +76,35 @@ the workspace initialization process. These hooks will not allow to commit the c
 
 Currently the following criteria are checked:
 
-- Rust code should always be formatted via `cargo fmt`.
-- Other code should always be formatted via `zk fmt`.
-- Dummy Prover should not be staged for commit (see below for the explanation).
-
-## Using Dummy Prover
-
-By default, the chosen prover is a "dummy" one, meaning that it doesn't actually compute proofs but rather uses mocks to
-avoid expensive computations in the development environment.
-
-To switch dummy prover to real prover, one must change `dummy_verifier` to `false` in `contracts.toml` for your env
-(most likely, `etc/env/base/contracts.toml`) and run `zk init` to redeploy smart contracts.
+- Code must be formatted via `zkstack dev fmt`.
+- Code must be linted via `zkstack dev lint`.
 
 ## Testing
 
-- Running the `rust` unit-tests:
-
-  ```
-  zk test rust
-  ```
-
-- Running a specific `rust` unit-test:
-
-  ```
-  zk test rust --package <package_name> --lib <mod>::tests::<test_fn_name>
-  # e.g. zk test rust --package zksync_core --lib eth_sender::tests::resend_each_block
-  ```
-
-- Running the integration test:
-
-  ```
-  zk server           # Has to be run in the 1st terminal
-  zk test i server    # Has to be run in the 2nd terminal
-  ```
+You can run tests using `zkstack dev test` subcommand. Just run:
 
-- Running the benchmarks:
-
-  ```
-  zk f cargo bench
-  ```
-
-- Running the loadtest:
+```bash
+zkstack dev test --help`
+```
 
-  ```
-  zk server # Has to be run in the 1st terminal
-  zk prover # Has to be run in the 2nd terminal if you want to use real prover, otherwise it's not required.
-  zk run loadtest # Has to be run in the 3rd terminal
-  ```
+to see all the options.
 
 ## Contracts
 
-### Re-build contracts
+### Build contracts
 
+Run:
+
+```bash
+zkstack dev contracts --help
 ```
-zk contract build
-```
+
+to see all the options.
 
 ### Publish source code on etherscan
 
+TODO: check the new way to do this
```

Comment:
Please give a hint how to use verify by default. 
 
https://github.com/matter-labs/zksync-era/blob/5092031050b30c39107df788317a15eaa921b136/zkstack_cli/crates/common/src/forge.rs#L269-L276


Or how to use forge verify
https://book.getfoundry.sh/reference/forge/forge-verify-contract

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3134/commits/26626de952ba959ba8bd9234e5c4a3bebdfa6882 and https://github.com/matter-labs/zksync-era/pull/3134/commits/d1f62a436404a89f6ed44f422c81df17af0263b1 (Please double check.)